hal_st/stm32fxxx/DmaStm.cpp

```diff
@@ -489,6 +631,9 @@ namespace hal
         auto streamRegister = DmaChannel[dmaIndex][streamIndex];
 #if defined(STM32F7) || defined(STM32F4)
         streamRegister->CR |= DMA_SxCR_CIRC;
+#elif defined(STM32WBA)
+        EnableHalfTransferCompleteInterrupt();
+        EnableTransferCompleteInterrupt();
```

Comment:
why was this added?

Reply:
I can remove this from now, it's part of the next PR to enable the circular mode. I'm still working on this.